### PR TITLE
Token page fixes

### PR DESCRIPTION
**Pull request by dariuszsobkowicz**
- fix background
- add search input

### CODE REVIEW

**Review by vishakh — APPROVED** [review]

**Comment by navi-arneja** [comment]
- [x] User should be able to get to the tokens landing page by selecting the “tokens” tab. Instead of the separate tab called “tokens page,” add a chevron expand icon to each tab that can be opened. [specified in designs] 

- [x] Wrapped Tezos should not show up on the side nav unless the user has a balance.

- [x] The balance for a token should be 16px from the bottom on all cards. 

- [x] Clicking on the token cards should lead to the relevant token page in Galleon.

- [x] Remove “Supported Tokens” if the user has no tokens. We only need this when there needs to be a distinction between “Your Tokens” and “Supported tokens”

- [x] Add a token card hover state. [specified in designs]

- [x] Ensure sizes of the same time of token card are equal.
 
- [x] "Your Tokens" should not be replicated in the "Supported Tokens" section. 

Designs and annotations: https://www.figma.com/file/KyMmhh4XU31uzaPO1oTvwY/Tokens-%26-Metadata?node-id=226%3A135

**Review by anonymoussprocket — APPROVED** [review]

File: src/containers/TokensPage/style.ts [hunk 1]
```diff
@@ -55,6 +55,15 @@ export const BoxBg = styled.div`
     cursor: pointer;
 `;
 
+export const BoxesGrid = styled.div`
+    display: grid;
+    overflow: hidden;
+    grid-template-columns: repeat(3, 1fr);
```
Comment (navi-arneja) [inline src/containers/TokensPage/style.ts:61]: When trying out fixes for the spacing, making this 4 columns filled up the space nicely and it still seemed reasonably responsive.

**Review by navi-arneja — COMMENTED** [review]
- [x] Update the number of columns of token cards to fix the spacing
- [x] Offset the shadow for the token card hover state 
- [x] Selecting the token card leads to its token page within Galleon. Selecting the hyperlink in the token card description links to an external site

Annotations for this review: https://www.figma.com/file/KyMmhh4XU31uzaPO1oTvwY/Tokens-and-Metadata?node-id=273%3A189

**Comment by vishakh** [comment]
All tasks done.

**Pull request merged**